src/connector.rs

```diff
@@ -31,6 +31,13 @@ impl<T> fmt::Debug for HttpsConnector<T> {
     }
 }
 
+impl HttpsConnector<()> {
```

Comment:
This seems unrelated to the acceptor changes? I'd prefer to have this PR focused on the acceptor.

Reply:
Fine, let's keep the `AcceptorBuilder::new()`. I still don't think we should add this here, since the `()` type argument makes this a little weird and unergonomic.